components/resourceDetails/resourceSharing/agentAccess/index.jsx

```diff
@@ -132,48 +123,11 @@ export default function AgentAccess({
   const onSubmit = submitHandler(
     authenticatedWebId,
     webId,
-    setOpen,
```

Comment:
I think this should still be here? `submitHandler` still takes that parameter, at least.